Experiment_Launcher_code/ExperimentLauncher.py

```diff
@@ -11,25 +11,26 @@ def main():
 
     if settings:
         # Extract the necessary parameters from the settings
+        experiment_name = settings.get('experiment_name')
+        comport_name = settings.get('comport_name')
```

Comment:
i dont understand completely why we must specify the com port

Reply:
Its an issue with windows, it assigns names to COM ports based on the current configuration of the OS. The COM port name changes from one machine to another and may change when you disconnect and reconnect Arduino, reset your lab computer, or whatever.  Its better to have to option to change it in the gui otherwise you will have to change it in the code every now and then. I added a default to the gui entry field. So as long as it does not change, the experimenter can simply ignore it. if it does change we have an easy way to fix it.

Reply:
ok